active/fellowship.md

```diff
@@ -2,30 +2,67 @@
 
 ## Scope of responsibilities
 
-The Fellowship Working Group manages the operation of the [Django Fellowship Program](https://www.djangoproject.com/fundraising/#fellowship-program). They select candidates to fill the fellowship, and act as line managers during the operation of the program.
+The Fellowship Working Group manages the [Django Fellowship Program](https://www.djangoproject.com/fundraising/#fellowship-program). This includes:
+
+- Interviewing candidates for open Fellowship slots and making hiring recommendations to the Board. This includes developing and maintaining interview and selection scripts, exercises, and rubrics.
+- Along with the Fellows themselves, maintaining the job description, outline of duties, and any other similar process documentation for the Fellow role (a "job manual", if you will). By and large these don't currently (as of Spring 2024), so the first group operating under this new charter will be responsible for starting to create this documentation. The Fellows and WG can maintain and update these without explicit Board approval, but should report major changes to the Board for review.
+- Designating one member of the committee to act as the line manager for active fellows. See [Fellow Management](#fellow-management), below, for details of this role.
+- Acting as an HR committee for Fellows. In the event of a complaint about a Fellow, or if there are performance issues, the Fellowship-WG will investigate, in concert with the [Code of Conduct WG](code-of-conduct.md) if necessary, and present findings and a recommendation to the board for action.
```

Comment:
Do we want the Fellowship WG to have any mediation responsibilities in case a Fellow wants to report an issue with an individual in a position of responsibility (board/steering/R+T/accessibility/the other Fellow) or do we expect all of this to go directly to the CoC WG to handle? (cc @jefftriplett)

I think it might be nice to explicitly have that the Fellowship WG as a HR committee would also function to support the Fellows rather than only for complaints and performance issues.

Note: I'm hoping none of this will be needed but think it's valuable to define 🙂

Reply:
> Do we want the Fellowship WG to have any mediation responsibilities in case a Fellow wants to report an issue with an individual in a position of responsibility (board/steering/R+T/accessibility/the other Fellow) or do we expect all of this to go directly to the CoC WG to handle? (cc @jefftriplett)

Yes, the Code of Conduct WG exists to help with these types of issues. I think it's important for us to note individual members more publicly in case someone in the WG (or board liaison) has a conflict and a reporter doesn't accident issue a complaint against someone who might also receive it. (this is more of a side note/tangent but worth noting because we have a lot of potential overlap here) 

> I think it might be nice to explicitly have that the Fellowship WG as a HR committee would also function to support the Fellows rather than only for complaints and performance issues.

> - Acting as an HR committee for Fellows. In the event of a complaint about a Fellow, or if there are performance issues, the Fellowship-WG will investigate, in concert with the [Code of Conduct WG](code-of-conduct.md) if necessary, and present findings and a recommendation to the board for action.

I think it's **very** important to point out that the CoC WG does not function in any HR role or capacity. With that in mind, I would consider splitting this bullet point so that it is abundantly clear. 

> - Acting as an HR committee for Fellows. 

> - In the event of a complaint about a Fellow, or if there are performance issues, the Fellowship-WG will investigate, in concert with the [Code of Conduct WG](code-of-conduct.md) if necessary, and present findings and a recommendation to the board for action.

Reply:
I had _intended_ the second sentence there ("In the event …") be a definition of what "acting as HR" means in this context, so splitting them into multiple points there leaves the definition of what "acting as HR" would mean. But now that I'm looking at this, I think it means we can just drop the "acting as HR" point entirely, just leaving "in the event of a complaint..." point.

@sarahboyce to your point, as I see it if a Fellow wants to make a report against someone in a position of responsibility, the process there is no different than for any other community member: they can go to the CoC committee. I suppose they can _also_ talk to the manager or WG or the Board, but at the moment there's any sort of conduct process to be instigated, I think it's vital to the community that it be handled by the CoC committee, as any other complaint might be. I _absolutely_ don't want to set up a situation where Fellows have additional steps before they can report something to the CoC WG. We could spell this out here if you think we should?

Reply:
Thank you both, I think this is much clearer in my head now. I will reiterate my understanding 👍 

If someone believes a community member (including Fellows) has breached the CoC, this should be reported to the CoC WG.
A line manager would normally want to be informed of these things and so the CoC WG might inform the line manager. But the reporting mechanism is still to the same as any other community member and the Fellows use this like anybody else.

If the concern is not around CoC but around performance[1], for the Fellows this should be reported to the line manager (or Fellow WG). As the Fellows are in paid positions, their performance has an added layer of accountability (including public reports and ways to handle complaints) as funds are finite and saying yes to one thing is saying no to something else.

If there are concerns around the performance of community members in voluntary unpaid positions, the situation is different and more nuanced. If someone wants to report, they should probably inform the board or a board member.

In some ways this is a little related to Jacob's comment
> Eventually, this probably should be a paid ED, if/when we get there 😄

If that sounds right then something like:
> In the event of a complaint about a Fellow, if this is a failure to fulfil the responsibilities of the position or around performance, this should be reported to the Fellowship-WG to investigate. The Fellowship-WG will present findings and a recommendation to the board for action. If this is a Code of Conduct issue, this should be reported to the [Code of Conduct WG](code-of-conduct.md) who will involve the Fellow WG if appropriate.

But probably worded better 🤔 
----

[1] I am defining performance to mean "fulfilment of the assigned responsibilities of the role"

Reply:
I share some of the concerns laid out here, to me this bullet point could lead to some confusion in terms of what an HR committee is and the scope of its function.

I think Sarah's proposal makes it clearer, assuming that is what you meant @jacobian ?